plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestingClickHouseServer.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.clickhouse;
+
+import org.testcontainers.containers.ClickHouseContainer;
+
+import java.io.Closeable;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+import java.util.Properties;
+
+import static java.lang.String.format;
+import static org.testcontainers.containers.ClickHouseContainer.HTTP_PORT;
+
+public class TestingClickHouseServer
+        implements Closeable
+{
+    private static final String CLICKHOUSE_IMAGE = "yandex/clickhouse-server:20.8";
+    private final ClickHouseContainer dockerContainer;
+
+    public TestingClickHouseServer()
+    {
+        // Use 2nd stable version
```

Comment:
what does "2nd stable " stand for ?

Reply:
I means use the second updated clickhouse stable version